utils/src/test/java/se/fortnox/reactivewizard/util/FluxTest.java

```diff
@@ -0,0 +1,27 @@
+package se.fortnox.reactivewizard.util;
```

Comment:
This class was just a local test i guess that got committed by accident

Reply:
Correct, removed.